src/util/db.rs

```diff
@@ -169,6 +169,17 @@ impl FromRow<u64> for u64 {
     }
 }
 
+impl FromRow<(u64, u64)> for (u64, u64) {
```

Comment:
If this is only ever used in the mempool, can you move it to `mempool.rs`?  From a code organization standpoint, `db.rs` is used for implementations that are used by multiple different databases; the `FromRow` / `FromColumn` implementations for subsystem-specific structs (which `(u64, u64)` appears to be) go into the modules that use them.